overlays/bootstrap.nix

```diff
@@ -106,6 +106,7 @@ in {
                 ++ self.lib.optional (version == "8.6.4") ./patches/ghc/ghc-8.6.4-reinstallable-lib-ghc.patch
                 ++ self.lib.optional (version == "8.6.5") ./patches/ghc/ghc-8.6.5-reinstallable-lib-ghc.patch
                 ++ self.lib.optional (version == "8.6.5") ./patches/ghc/ghc-8.6.5-atomic-arm-arch.patch
+                ++ self.lib.optional (version == "8.6.5") ./patches/ghc/MR555-writable-rel-ro-data.patch
```

Comment:
The naming seems of. MR refers to ghc-upstream merge request [MR555](http://gitlab.haskell.org/ghc/ghc/-/merge_requests/555), is not this :-)

You might want to open a MR for this actually. But calling it MRxxx will suggest there is an upstream mergerequest for this.

Reply:
Oops, sorry! I will do so then, and then update this MR :-)